src/store/middleware/driveMiddleware.js

```diff
@@ -37,6 +37,33 @@ const driveMiddleware = store => next => action => {
       break;
     }
 
+    case requestTurnInPlaceDrive.type: {
+      if (selectMotorsAreEnabled(store.getState())) {
+        const { steer } = action.payload;
+        store.dispatch(messageRover({
+          message: {
+            type: "turnInPlaceDriveRequest",
+            steer
+          }
+        }));
+        break;
```

Comment:
You need to move this break out of the if scope into the case scope. If the selectMotorsAreEnabled is false, then it won't break, and the next will be ran